src/main/java/io/jenkins/plugins/synopsys/security/scan/SecurityScanner.java

```diff
@@ -46,7 +47,8 @@ public int runScanner(Map<String, Object> scanParams, FilePath bridgeInstallatio
             throws PluginExceptionHandler {
         int scanner = -1;
 
-        List<String> commandLineArgs = scannerArgumentService.getCommandLineArgs(scanParams, bridgeInstallationPath);
+        List<String> commandLineArgs = scannerArgumentService.getCommandLineArgs(
+                Utility.installedBranchSourceDependcies(), scanParams, bridgeInstallationPath);
```

Comment:
Do we need to pass this new parameter all the way to `SCMReposirotyService` class? Or can it be directly used in `fetchSCMRepositoryDetails` method of `SCMReposirotyService` class?

Reply:
Yes, otherwise you would have to mock the implementation of `installedBranchSourceDependcies()`. This is not possible as it uses methods from `Jenkins API`.